yoga/include/YGEnums.h

```diff
@@ -0,0 +1 @@
+../YGEnums.h
```

Comment:
New files need the copyright header present in the other header files.

Reply:
OK, copyright added to new files.